### DISCUSSION TITLE
`rose-pine-moon` and `rose-pine-dawn` don't highlight selected text

### DISCUSSION THREAD

**Discussion by pavelzw** [ghostty-org/ghostty#3730] [Feature Requests, Ideas] [answered]
Both rose-pine-moon and rose-pine-dawn don't highlight selected text:

rose-pine-dawn:
![image](https://github.com/user-attachments/assets/2e9d3ad7-4cf6-4bb7-a055-3549e9d06366)

catppuccin-latte works:
![image](https://github.com/user-attachments/assets/7554807a-d31a-40cb-80cb-a431bc2995d9)

is this intended?

**Comment by rhodes-b** [comment]
This is the current indented behavior of Ghostty. 

I have seem someone else request this as a feature to match similar to selecting text while in Visual mode although im not sure if they ever made a disucssion here about it

**Comment by 00-kat** [comment]
Upstream bug https://github.com/mbadolato/iTerm2-Color-Schemes/issues/439 fixed in https://github.com/mbadolato/iTerm2-Color-Schemes/pull/485 which will be pulled in soon (but probably not in time for v1.0.1).

Workaround in https://github.com/rose-pine/ghostty?tab=readme-ov-file#usage.

**Reply by 00-kat** [reply, marked as answer]
Fixed in #4152.

**Comment by pavelzw** [comment]
fixed with the latest nightly release
